libcodechecker/cmd/cmd_line_client.py

```diff
@@ -356,6 +357,50 @@ def get_report_data(client, reports, file_cache):
         return {'files': file_sources,
                 'reports': report_data}
 
+    def reports2report_data(reports):
+        """
+        Converts reports from Report class from one plist file
+        to report data events for the HTML plist parser.
+        """
+        file_sources = {}
+        fname_to_fid = {}
+        report_data = []
+        findex = 0
+
+        for report in reports:
+            # Not all report in this list may refer to the same files
+            # thus we need to create a single file list with
+            # all files from all reports.
+            for f in report.files:
```

Comment:
You could use enumerate here, so you will not need a separate `findex` variable.

```
for index, f in enumerate(report.files):
...
```
The same applies for the other `index` variable below.

Reply:
I prefer the old fashion way. ;)